book/src/4.teams/2.manifest.md

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
intentionally empty?

Reply:
Yeah, it's in draft (you can do drafts in mddocs)